editor/index.js

```diff
@@ -49,17 +50,29 @@ window.jQuery( document ).on( 'heartbeat-tick', ( event, response ) => {
  * The return value of this function is not necessary if we change where we
  * call createEditorInstance(). This is due to metaBox timing.
  *
- * @param {String}  id       Unique identifier for editor instance
- * @param {Object}  post     API entity for post to edit
- * @param {?Object} settings Editor settings object
+ * @param {String}  id           Unique identifier for editor instance
+ * @param {Object}  post         API entity for post to edit
+ * @param {?Object} settings     Editor settings object
+ * @param {?Object} initialState Initial editor state to hydrate
  * @return {Object} Editor interface. Currently supports metabox initialization.
  */
-export function createEditorInstance( id, post, settings ) {
+export function createEditorInstance( id, post, settings, initialState ) {
 	const target = document.getElementById( id );
 
+	// When an unhandled error occurs, user can choose to reboot the editor,
+	// replacing a previous mount using initialState from the crashed state.
+	unmountComponentAtNode( target );
```

Comment:
I noticed that we don't need to call `unmountComponentAtNode` when no `initialState` is passed. Not sure if we really need to change that.

Reply:
>I noticed that we don't need to call `unmountComponentAtNode` when no `initialState` is passed. Not sure if we really need to change that.

Yeah, it doesn't harm anything to keep it this way, but could make things clearer if the condition were to demonstrate it only occurs when rebooting.